### DISCUSSION TITLE
谷歌网盘一直出现以下报错

### DISCUSSION THREAD

**Discussion by 10YuJi** [AlistGo/alist#2451] [Q&A]
Failed init storage but storage is already created: google_drive.Addition.ChunkSize: readUint64: unexpected character: �, error found in #10 byte of ...|nk_size":"5"}|..., bigger context ...|OCSPX-3bejJ3lEAu7vsqBi3Aou45opxd8O","chunk_size":"5"}|...

**Comment by 10YuJi** [comment]
可能是3.4的问题，更新就没事了。

**Comment by HsinKwok** [comment]
在存储列表里面编辑。重新保存一下就好了。
